app.rb

```diff
@@ -1,9 +1,32 @@
 require 'sinatra/base'
+require './lib/message'
 
 class Chitter < Sinatra::Base
+
+  get '/' do
```

Comment:
Here, you could render a simple template with en "Enter" button so that the user can get from this page to the next.  At the moment, they'd get stuck! :)